README.md

```diff
@@ -75,7 +76,13 @@ Shows details on when product will be in stock next.
 
 ### Heroku Deployment
 
-This repo includes a Procfile that can be used to deploy to heroku.
+This repo includes a `heroku.yml` that can be used to deploy to heroku.
+
+Because this application is using Docker, before pushing any code to Heroku, set the heroku stack to use container instead of Ubuntu.
```

Comment:
```suggestion
Because this application is using Docker, before pushing any code to Heroku, set the heroku stack to use container instead of runtime.
```

Reply:
👍